src/components/MARCTable/MARCTableRow.js

```diff
@@ -559,6 +588,10 @@ MARCTableRow.propTypes = {
   isFirst: PropTypes.bool,
   isLast: PropTypes.bool,
   isSubline: PropTypes.bool,
+  subfieldsData: PropTypes.arrayOf(PropTypes.object),
```

Comment:
Consider using `PropTypes.shape` instead of `PropTypes.object` to understand the structure of an object and its required fields.

Reply:
done